components/calendly_v2/actions/get-event/get-event.mjs

```diff
@@ -14,21 +25,37 @@ export default {
     },
     event_uuid: {
       type: "string",
-      description: "The event's unique identifier.",
+      label: "Event UUID",
+      description:
+        "The event's unique identifier.",
+      optional: true,
+    },
+    event_url: {
+      type: "string",
+      label: "Event URL",
+      description: "The URL of the event to retrieve information about. If you are using a Calendly Source in the same workflow, you would use *{{steps.trigger.event.payload.event}}*.",
+      optional: true,
     },
   },
-  async run({ $ }) {
-  //See the API docs: https://developer.calendly.com/docs/api-docs/reference/calendly-api/openapi.yaml/paths/~1scheduled_events~1%7Buuid%7D/get
 
-    if (!this.event_uuid) {
-      throw new Error("Must provide event_uuid parameter.");
+  async run({ $ }) {
+    if (!this.event_uuid && !this.event_url) {
+      throw new ConfigurationError(
+        "Error: You must provide either Event UUID or Event URL.",
+      );
     }
 
-    return await axios($, {
-      url: `https://api.calendly.com/scheduled_events/${this.event_uuid}`,
+    const event_uuid = this.event_uuid || getEventUuidFromUrl(this.event_url);
+
+    const response = await axios($, {
+      url: `https://api.calendly.com/scheduled_events/${event_uuid}`,
       headers: {
         Authorization: `Bearer ${this.calendly_v2.$auth.oauth_access_token}`,
       },
     });
+
+    $.export("$summary", "Successfully retrieved event.");
```

Comment:
@malexanderlim bonus points for including the event name (or something else that is useful) in the summary message here. https://pipedream.com/docs/components/guidelines/#use-summary-to-summarize-what-happened